fbw-a380x/src/systems/systems-host/systems/tcas/index.ts

```diff
@@ -0,0 +1,3 @@
+import { LegacyTcasComputer } from './components/LegacyTcasComputer';
+
+export { LegacyTcasComputer };
```

Comment:
Why do we even need a barrel for this?

Reply:
Artifact of the ancient days where we compiled this as a separate module, I agree it's silly but I don't want to spend more time refactoring at this point.